examples/messengers/telegram/7_conditions_with_media.py

```diff
@@ -0,0 +1,168 @@
+# %% [markdown]
+"""
+# 7. Conditions with Media
+
+This example shows how to use media-related logic in your script.
+"""
+
+# %%
+import os
+
+import dff.script.conditions as cnd
+from dff.script import Context, Actor, TRANSITIONS, RESPONSE
+
+from telebot import types
+
+from dff.messengers.telegram import (
+    PollingTelegramInterface,
+    TelegramMessenger,
+    update_processing_service,
+)
+from dff.pipeline import Pipeline
+from dff.script.responses.generics import Response, Image, Attachments
+from dff.utils.testing.common import is_interactive_mode
+
+
+# %%
+# kitten picture info:
+kitten_id = "Y0WXj3xqJz0"
+kitten_ixid = "MnwxMjA3fDB8MXxhbGx8fHx8fHx8fHwxNjY4NjA2NTI0"
+kitten_width = 640
+kitten_url = (
+    f"https://unsplash.com/photos/"
+    f"{kitten_id}/download?ixid={kitten_ixid}"
+    f"&force=true&w={kitten_width}"
+)
+
+
+# %% [markdown]
+"""
+To filter user messages depending on whether or not media files were sent,
+you can use the `content_types` parameter of the `message_handler`.
+
+If you want to access properties of some specific file, you can get
+the message from the context:
+
+.. code-block:: python
+
+    message = ctx.last_request
+
+The files will then be accessible as properties: message.photo, etc.
+
+"""
+
+
+# %%
+def doc_is_photo(message: types.Message):
+    return message.document and message.document.mime_type == "image/jpeg"
+
+
+# %%
+messenger = TelegramMessenger(os.getenv("TG_BOT_TOKEN", "SOMETOKEN"))
+
+
+# %%
+script = {
+    "root": {
+        "start": {RESPONSE: Response(text=""), TRANSITIONS: {("pics", "ask_picture"): cnd.true()}},
+        "fallback": {
+            RESPONSE: "Finishing test, send /restart command to restart",
+            TRANSITIONS: {
+                ("pics", "ask_picture"): messenger.cnd.message_handler(
+                    commands=["start", "restart"]
+                )
+            },
+        },
+    },
+    "pics": {
+        "ask_picture": {
+            RESPONSE: Response(text="Send me a picture"),
+            TRANSITIONS: {
+                ("pics", "send_one", 1.1): cnd.any(
+                    [
+                        messenger.cnd.message_handler(content_types=["photo"]),
```

Comment:
it should be better to add description of differences between this two methods of  response type checking